binsync/common/ui/force_push/panels/functions_table.py

```diff
@@ -284,18 +284,14 @@ def handle_filteredit_change(self, text):
         else:
             self.select_all.setChecked(True)
 
-
     def push(self):
         self.proxymodel.setFilterFixedString("")
         for i in range(self.proxymodel.rowCount()):
-
             proxyIndex = self.proxymodel.index(i, 0, QModelIndex())
             mappedIndex = self.proxymodel.mapToSource(proxyIndex)
             if self.model.checkState(mappedIndex):
                 func_addr = int(self.model.data(mappedIndex), 16)
-                success = self.controller.force_push_function(func_addr)
-                l.info(f"Pushing function {hex(func_addr)} {'was Successful' if success else 'Failed'}")
```

Comment:
I moved the logging statements to the controller so that we can log info when a force push is completed regardless of if it is invoked from force push ui or force push context